### DISCUSSION TITLE
Rerunning only one of the tests

### DISCUSSION THREAD

**Discussion by arupc-qcom** [riscv-software-src/riscv-perf-model#156] [Q&A]
Invoking `make regress` runs all the tests in the regression.

Is there way to rerun just one of the tests?

**Comment by aarongchan** [comment]
To test individually, I usually go to the test directory for that submodule, run `make`, and then I can run an individual test from there.
Ex:
```
cd test/core/issue_queue
make
./IssueQueue_test small_core.out -c test_cores/test_big_core.yaml --input-file test_int_pipe.json
```

Not sure if there's an easier way to do this, such as being able to call a specific test from outside test folder.

**Comment by klingaard** [comment]
`ctest -V -R <testname>` is one way.  There are other flags you can use with cmake to make it simpler
